### DISCUSSION TITLE
GitHub Search API returns permission error for a user that is part of our org

### DISCUSSION THREAD

**Discussion by vicentllongo** [community/community#33906] [Apps, API and Webhooks] [answered]
Hi, since a few days ago the following Search API query has started returning the error below for one of our organization users. The same query works fine for all the other users that are part of the organization. Any idea what could be the reason? The user is part of the organization and as far as we know no changes have been done in the GitHub user profile. I also found [this discussion](https://github.com/orgs/community/discussions/24572) that seems related, but in our case the user does have public repositories.

```
https://api.github.com/search/issues?q=is:open is:issue+org:Leadfeeder+assignee:Darren-LF
```

```json
{
    "message": "Validation Failed",
    "errors": [
        {
            "message": "The listed users cannot be searched either because the users do not exist or you do not have permission to view the users.",
            "resource": "Search",
            "field": "q",
            "code": "invalid"
        }
    ],
    "documentation_url": "https://docs.github.com/v3/search/"
}
```

**Comment by byrneh** [comment, marked as answer]
Was this performed from command line, if so what scope permissions does the user have with the PAT used for autthentication?
WHat resspone do they get entering the same comand in the browser URL bar (with curerntly signed in user session)?

**Reply by vicentllongo** [reply]
Thank you for your response. I checked again after a few days and the error doesn't appear anymore. I guess it was a temporary glitch :)
